jest.config.js

```diff
@@ -1,16 +1,16 @@
 module.exports = {
   projects: [
     '<rootDir>/packages/core',
-    '<rootDir>/e2e\\core-e2e',
```

Comment:
Inconsistent slashes can cause chaos, but I don't think it was currently causing issues.